hdk-rust/src/api/entry_type_properties.rs

```diff
@@ -0,0 +1,27 @@
+use meta::ZomeDefinition;
+use holochain_core_types::entry::entry_type::EntryType;
+use error::{ZomeApiResult, ZomeApiError};
+use holochain_json_api::json::JsonString;
+
+#[allow(improper_ctypes)]
+extern "C" {
+    fn zome_setup(zd: &mut ZomeDefinition);
+}
+
+// Returns the properties defined with an entry type
+// It is encouraged to using JSON to encode structured properties
+// with an entry
+pub fn entry_type_properties(name: &EntryType) -> ZomeApiResult<JsonString> {
+    let mut zd = ZomeDefinition::new();
+    unsafe { zome_setup(&mut zd) };
+
+    let entry_def = zd.entry_types.iter().find(|elem| {
```

Comment:
Is iterating through the zome def the best way to find the given entry? Not sure about this one